### PR TITLE
New Tests

### PR DESCRIPTION
**Pull request by MarynaLawson**

### CODE REVIEW

File: src/test/java/school/redrover/BugsBustersTest.java [hunk 1]
```diff
@@ -300,4 +300,31 @@ public void testMariaVerifyLogoHome() {
...
+    public void testMarynaLBuy() {
+        getDriver().get("https://nightowlsp.com/");
+
+        WebElement wireless = getDriver().findElement(By.xpath("//*[@id='maincontent']/div[2]/div/div[4]/div/div/div/div[2]/a"));
+        wireless.click();
+
+        WebElement shopAllWirelessSystem = getDriver().findElement(By.xpath("//*[@id='maincontent']/div[2]/div/div[3]/div[2]/a"));
+        shopAllWirelessSystem.click();
+
+        WebElement addToCart = getDriver().findElement(By.xpath("//*[@id='product-item-info_836']/div[2]/div[2]/div[2]/div/div[1]/form/button/span"));
```
Comment (AliMariKat) [inline src/test/java/school/redrover/BugsBustersTest.java:323]: try this shorter xpath: //a[@href='/product/wireless-security.html']

File: src/test/java/school/redrover/BugsBustersTest.java [hunk 2]
```diff
@@ -300,4 +300,31 @@ public void testMariaVerifyLogoHome() {
...
+        String title = getDriver().getTitle();
+
+        Assert.assertEquals(title, "Free Invoice Generator");
+    }
+
+    @Test
+    public void testMarynaLBuy() {
+        getDriver().get("https://nightowlsp.com/");
+
+        WebElement wireless = getDriver().findElement(By.xpath("//*[@id='maincontent']/div[2]/div/div[4]/div/div/div/div[2]/a"));
```
Comment (AliMariKat) [inline src/test/java/school/redrover/BugsBustersTest.java:317]: Try this shorter xpath:    //a[@href='/product/wireless-security.html']

File: src/test/java/school/redrover/BugsBustersTest.java (hunk 1 above)
Comment (AliMariKat) [inline src/test/java/school/redrover/BugsBustersTest.java:323]: //a[@class='btn-link btn primary']

File: src/test/java/school/redrover/BugsBustersTest.java [hunk 3]
```diff
@@ -300,4 +300,31 @@ public void testMariaVerifyLogoHome() {
...
+        wireless.click();
+
+        WebElement shopAllWirelessSystem = getDriver().findElement(By.xpath("//*[@id='maincontent']/div[2]/div/div[3]/div[2]/a"));
```
Comment (AliMariKat) [inline src/test/java/school/redrover/BugsBustersTest.java:320]: //a[@class='btn-link btn primary']

**Review by AliMariKat — APPROVED** [review]

**Review by GelikaUS — APPROVED** [review]

**Pull request merged**